lib/content-services/src/lib/common/services/saved-searches-legacy.service.spec.ts

```diff
@@ -0,0 +1,156 @@
+/*!
+ * @license
+ * Copyright © 2005-2025 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { AlfrescoApiService } from '../../services/alfresco-api.service';
+import { NodeEntry } from '@alfresco/js-api';
+import { AlfrescoApiServiceMock } from '../../mock';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
```

Comment:
HttpClientTestingModule is deprecated - can you use something different?
<img width="872" height="483" alt="image" src="https://github.com/user-attachments/assets/e9529256-de2b-4e56-9b0a-8278ffed2498" />